.github/workflows/deploy.yml

```
@@ -46,20 +46,43 @@ jobs:
+        echo "Pulling puzzle data from DVC..."
+        cd scripts/datascience/themes_quality
+        dvc pull puzzle_generation_output.dvc
+        echo "DVC puzzle data pulled successfully"
+        echo "Available puzzle files:"
+        ls -la puzzle_generation_output/
+
+    - name: Run Database Migration - Upload Gemini Puzzles
+      run: |
+        echo "Checking for DVC puzzle data..."
+        DVC_PUZZLE_FILE="../scripts/puzzle-generation/gemini-puzzles_firebase.json"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)